chia/wallet/puzzles/custody/restriction_puzzles/wrappers/force_assert_coin_announcement.clsp

```diff
@@ -0,0 +1,16 @@
+(mod
+  (
+    Conditions
+  )
+
+  (include condition_codes.clib)
+
+  (defun check_conditions (conditions)
+    (if (= (f (f conditions)) ASSERT_COIN_ANNOUNCEMENT)
```

Comment:
I understand that this is failing if `(f (f conditions))` is called when `conditions` is empty, but might be worth adding a comment explaining this

Reply:
done